kythe/cxx/extractor/testdata/has_include_test.cc

```diff
@@ -63,7 +63,7 @@ argument: "-I./kythe/cxx/extractor"
 argument: "./kythe/cxx/extractor/testdata/has_include.cc"
 argument: "-fsyntax-only"
 source_file: "./kythe/cxx/extractor/testdata/has_include.cc"
-working_directory: "TEST_CWD"
```

Comment:
Are there any tests left which test the "non-stable root" path?

Reply:
I'll add a test of the new logic.  None of the existing tests tested for the "non-stable root" because it was unstable, so they set it to a well-known value (`TEST_CWD`) and then used `set_working_directory` on the extracted protobuf.

So there are exactly as many tests of the previous behavior as there used to be?